java/odsa-spring/src/main/java/com/oracle/dev/jdbc/odsa/spring/EmployeeNotFoundException.java

```diff
@@ -0,0 +1,31 @@
+/*
+  Copyright (c) 2021, 2022, Oracle and/or its affiliates.
+
+  This software is dual-licensed to you under the Universal Permissive License
+  (UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl or Apache License
+  2.0 as shown at http://www.apache.org/licenses/LICENSE-2.0. You may choose
+  either license.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package com.oracle.dev.jdbc.odsa.spring;
+
+class EmployeeNotFoundException extends RuntimeException {
+
+  private static final long serialVersionUID = 7768215094663842819L;
```

Comment:
Please check if this is needed.

Reply:
@Kuassim please review, approve and notify Jean per the process (2nd level approval). Thanks!

Reply:
Approved; up to @jeandelavarene